src/Components/Card/index.tsx

```diff
@@ -63,15 +63,23 @@ const Card = ({ }) => {
     const neumorphicShadow = hass.themes.darkMode ? '-5px -5px 8px rgba(50, 50, 50,.2),5px 5px 8px rgba(0,0,0,.08)' : '-4px -4px 8px rgba(255,255,255,.5),5px 5px 8px rgba(0,0,0,.03)'
     const defaultShadow = 'var( --ha-card-box-shadow, 0px 2px 1px -1px rgba(0, 0, 0, 0.2), 0px 1px 1px 0px rgba(0, 0, 0, 0.14), 0px 1px 3px 0px rgba(0, 0, 0, 0.12) )'
 
-    const hidden = state.toLowerCase() !== 'printing' && !hiddenOverride;
+    const lowerState = state.toLowerCase();
+    const hidden = 
+        hiddenOverride ?
+            false  // respect 'always_show' parameter
+            : config.states_show && config.states_show.indexOf(lowerState) > -1 ?
```

Comment:
wouldn't makes more sense to make state case sensitive for both show/hide states, Octoprint states are capitalization which means if user copied the state from developer tools then it wouldn't works unless it manually lower-case it

```suggestion
            : config.states_show && config.states_show.indexOf(state) > -1 ?
```